packages/flex-dev-utils/src/packages/lib/getLatestFlexUIVersion.ts

```diff
@@ -0,0 +1,26 @@
+import semver from 'semver';
+
+import getRegistryVersion from './getRegistryVersion';
+
+export default async function getLatestFlexUIVersion(version: number): Promise<string> {
```

Comment:
Add a docblock please

Reply:
Seems better if you pass a string instead of a `number` and then you can convert it to a Semver and do whatever you want, for example, get the major version here.

Reply:
the number is only used to compare to the major version found by server so if it was a string we would have to convert it to a number for the comparison